src/collision.cpp

```diff
@@ -355,6 +290,151 @@ collisionMoveResult collisionMoveSimple(Environment *env, IGameDef *gamedef,
 			cinfo.emplace_back(true, 0, p, box);
 		}
 	}
+	return any_position_valid;
+}
+
+static void addObjectBoxes(Environment *env,
```

Comment:
1. snake_case
2. Did you touch any of these lines or were they only moved?

Reply:
I only moved them to not duplicate code.